_posts/2000-01-01-intro.md

```diff
@@ -5,6 +5,8 @@ color: black
 style: center
 ---
 
+<img class = "coala-logo" src="{{ site.logo }}">
```

Comment:
Awesome!! :tada:

but 
`class="coala-logo"`

sorry I'm so nitpicky :( and pretty much a human linter :sweat: